### ISSUE TITLE
Question about more datasets and more baselines

### ISSUE THREAD

**Issue by Dinxin** [snap-stanford/pretrain-gnns#12]
Why don't you give the associated experimental results on some regression tasks in the MoleculeNet benchmark? Additionally, there are some pertaining techniques tailored for molecule data such as mol2vec, seq2seq fingerprint. However, I don't see the model comparison.

**Comment by weihua916** [comment]
Great question! In this paper, we focus on GNNs. Testing all those techniques is beyond the scope of a single paper, but it is definitely an important work to do.

**Comment by badtom** [comment]
Given that you do a "graph-level multi-task supervised pre-training" on the preprocessed ChEMBL dataset from the publication "Large-scale comparison of machine learning methods for drug target prediction on ChEMBL" did you perhaps also check whether your GNN outperforms the multi-task feedforward neural network from this publication on this set (note that the best AUC ROC results for a random split were reported by the authors when using additional data normalization in http://www.bioinf.jku.at/research/lsc/report.pdf)?
